tests/conftest.py

```diff
@@ -1,28 +1,110 @@
-import fnmatch
-import http.client
+import asyncio
 import json
 import logging
 import os
 import random
 import shutil
-import subprocess
-from time import sleep
+import socket
+from copy import deepcopy
+from pathlib import Path
+from subprocess import STDOUT, Popen, run
 
 import pytest
-from hopr import HoprdAPI
 
-random_data = os.urandom(8)
-SEED = int.from_bytes(random_data, byteorder="big")
+from .node import Node
+
+SEED = int.from_bytes(os.urandom(8), byteorder="big")
 random.seed(SEED)
 
+# Global variables
 LOCALHOST = "127.0.0.1"
-OPEN_CHANNEL_FUNDING_VALUE = "1000000000000000000000"
+OPEN_CHANNEL_FUNDING_VALUE = 1000
 
 TICKET_AGGREGATION_THRESHOLD = 100
 TICKET_PRICE_PER_HOP = 100
 
+FIXTURES_PREFIX = "hopr-smoke-test"
+NODE_NAME_PREFIX = "hopr-smoke-test-node"
+
+NETWORK1 = "anvil-localhost"
+NETWORK2 = "anvil-localhost2"
+
+API_TOKEN = "e2e-API-token^^"
+PASSWORD = "e2e-test"
 
-def pytest_addoption(parser):
+PWD = Path(__file__).parent
+FIXTURES_DIR = Path("/tmp")
+
+ANVIL_CFG_FILE = FIXTURES_DIR.joinpath(f"{FIXTURES_PREFIX}-anvil.cfg")
+ANVIL_LOG_FILE = FIXTURES_DIR.joinpath(f"{FIXTURES_PREFIX}-anvil.log")
+
+PROTOCOL_CONFIG_FILE = PWD.parent.joinpath("scripts/protocol-config-anvil.json")
+TEST_PROTOCOL_CONFIG_FILE = FIXTURES_DIR.joinpath(f"{NODE_NAME_PREFIX}-protocol-config.json")
+DEPLOYMENTS_SUMMARY_FILE = PWD.parent.joinpath("ethereum/contracts/contracts-addresses.json")
+PREGENERATED_IDENTITIES_DIR = PWD.joinpath("identities")
+
+NODES = [
+    Node(
+        19091,
+        13301,
+        API_TOKEN,
+        FIXTURES_DIR.joinpath(f"{NODE_NAME_PREFIX}_1"),
+        "localhost",
+        NETWORK1,
+    ),
+    Node(
+        19092,
+        13302,
+        None,
```

Comment:
No API token?

Reply:
There was none before. Check what is on master.